docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -35,7 +35,7 @@ This can be the ip or the FDQN, the url scheme (``http://``) is optional.
 Configuration properties
 ------------------------
 
-The following configuration properties are available:
+General configuration properties:
```

Comment:
Let's change like this so that "General configuration properties" can go to under "Configuration properties".  It's also helpful for generating a link. 
```
General configuration properties
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```